net-misc/gns3-server/gns3-server-2.1.16.ebuild

```diff
@@ -0,0 +1,59 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{5,6} )
```

Comment:
How about 3.7?

Reply:
while gns3 claims to be 3.5+ many of its dependencies in tree are not 3.7 compatible yet